extensions/amp-bind/amp-bind.md

```diff
@@ -98,7 +98,7 @@ For AMP components, only select attributes are bindable:
 | Component | Attributes |
 | --- | --- |
 | amp-carousel | slide |
-| amp-img | src, srcset, alt |
+| amp-img | src, srcset, alt, height, width |
```

Comment:
We actually support [width] and [height] on all AMP extensions. Can you remove this from the table and maybe add a sentence above?